index.js

```diff
@@ -118,55 +115,59 @@ function around(index, lng, lat, maxResults, maxDistance, predicate) {
 }
 
 // lower bound for distance from a location to points inside a bounding box
-function boxDist(lng, lat, node, cosLat, sinLat) {
+function boxDist(lng, lat, node) {
     var minLng = node.minLng;
     var maxLng = node.maxLng;
     var minLat = node.minLat;
     var maxLat = node.maxLat;
 
     // query point is between minimum and maximum longitudes
     if (lng >= minLng && lng <= maxLng) {
-        if (lat <= minLat) return earthCircumference * (minLat - lat) / 360; // south
-        if (lat >= maxLat) return earthCircumference * (lat - maxLat) / 360; // north
-        return 0; // inside the bbox
+        var dLat = (lat < minLat) ? (lat - minLat) : ((lat > maxLat) ? (lat - maxLat) : 0);
+        return haverSin(dLat * rad);
```

Comment:
Probably nitpicking, but would it be faster to return `0` immediately instead of calculating `haverSin(0)`?